ci/cron/daily-compat.yml

```diff
@@ -173,6 +173,94 @@ jobs:
         env:
           GCRED: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
 
+  - job: perf_http_json_oracle
+    timeoutInMinutes: 120
+    pool:
+      name: "ubuntu_20_04"
+      demands: assignment -equals default
+    steps:
+      - checkout: self
+      - bash: ci/dev-env-install.sh
+        displayName: 'Build/Install the Developer Environment'
+      - bash: ci/configure-bazel.sh
+        displayName: 'Configure Bazel for root workspace'
+        env:
+          IS_FORK: $(System.PullRequest.IsFork)
+          # to upload to the bazel cache
+          GOOGLE_APPLICATION_CREDENTIALS_CONTENT: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
+      - template: ../bash-lib.yml
+        parameters:
+          var_name: bash_lib
+      - bash: |
+          set -euo pipefail
+          eval "$(dev-env/bin/dade assist)"
+          source $(bash_lib)
+
+          docker login --username "$DOCKER_LOGIN" --password "$DOCKER_PASSWORD"
+          IMAGE=$(cat ci/oracle_image)
+          docker pull $IMAGE
+          # Cleanup stray containers that might still be running from
+          # another build that didn’t get shut down cleanly.
+          docker rm -f oracle || true
```

Comment:
Is there a chance the `oracle` container is removed while another pipeline is using it?

Reply:
This logic is similar to what we use for our `Linux_oracle` jobs , as long as we don't run multiple jobs in parallel on the same azure boxes this should be fine IMO

Reply:
We only ever have one job running per node so that’s not an issue.

Reply:
Cool, thanks! 🙇🏻